src/bootloader/rollbacks/fiovb.h

```diff
@@ -39,7 +39,7 @@ class FiovbRollback : public Rollback {
     std::string sink;
     if (Utils::shell("fiovb_printenv bootfirmware_version", &sink) != 0) {
       LOG_WARNING << "Failed to read bootfirmware_version";
-      return;
+      sink = std::string();
```

Comment:
So, `sink` won't be empty if the fiovb command fails? Maybe it would be useful to print it before nulling if it contains some useful error messages.

Reply:
@Tim-Anderson did you check if sink ended up with an error there? I wonder if fiovb_printenv will dump the error to stderr instead.

Reply:
To clarify. The optee error "E/TA: ..." is output to the console only. The "READ_PVALUE failed: 0xffff0006 / 4"  and the "Cant print the environment" are both going to stderr so stdout will have no output.